### PR TITLE
Upgrade SQLAlchemy to 1.1

### PR DESCRIPTION
**Pull request by amercader**
After diving into this for quite a while I think this is ready for review. I don't think it adds more uncertainty to VDM's behaviour, and any issues we can pick up before the next release.

As usual when involving vdm, these are really small patches with huge commit messages for background but all commits should be straightforward to follow.

Most important one is 26dd7ad. The Tl;DR is that if in a single transaction the Session has recorded a change for the creation of an object and for the update of the same object, we will just record an activity for the creation of the object. @TkTech as you have worked with activities I'd like to make sure that you are happy with this (26dd7ad).

@wardi a85b684 is a a change on the trigger functions creation that you worked on recently.

And of course, VDM must go :hocho:

### CODE REVIEW

**Comment by TkTech** [comment]
@amercader more than happy with it, I think someone has actually implemented similar behaviour in our fork already since our clients complained.

File: ckan/logic/action/create.py [hunk 1]
```diff
@@ -610,7 +610,7 @@ def member_create(context, data_dict=None):
                               table_id=obj.id,
                               group_id=group.id,
                               state='active')
-
+        member.group = group
```
Comment (wardi) [inline ckan/logic/action/create.py]: is this an unrelated change?

File: ckan/migration/versions/016_uuids_everywhere.py [hunk 2]
```diff
@@ -166,19 +172,18 @@ def create_uuids(migrate_engine, primary_table_name, revision_table_name):
 
 def drop_sequencies(migrate_engine):
 
-    sequencies = ['package_extra', 'package_extra_revision', 'package',
-                  'package_resource', 'package_resource_revision',
-                  'package_revision',' package_tag', 'package_tag_revision',
+    sequencies = ['package_extra', 'package',
+                  'package_resource',
+                  'package_tag',
```
Comment (wardi) [inline ckan/migration/versions/016_uuids_everywhere.py]: looks like a change in behaviour. we'd still want to drop these sequences if they were created in an earlier migration, right?

File: ckan/logic/action/create.py (hunk 1 above)
Comment (amercader) [inline ckan/logic/action/create.py]: No, with the 1.1 upgrade not explicitly assigning the group to the new member object led to the relation between the two not being set at the database level, which caused loads of [test failures](https://circleci.com/gh/ckan/ckan/2679?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) due to Authorization errors

File: ckan/migration/versions/016_uuids_everywhere.py (hunk 2 above)
Comment (amercader) [inline ckan/migration/versions/016_uuids_everywhere.py]: You are right, fixed in dafa45b
